js/bootstrap-datepicker.js

```diff
@@ -2034,3 +2038,13 @@
 	});
 
 }));
+
+function deprecate(msg) {
```

Comment:
I've tried to move the function to `$.fn.datepicker.deprecate` but then I can't use the function in the alias method. So all tests fail, so where should I declare the method?